netdata/netdata
# Claiming issues

#### Summary

Cleaning up and polishing some aspects of claiming.
This addresses the requirements in product/595, product/535.
Closes #8486 

##### Component Name
claiming
cloud

##### Test Plan
Verify that this works in the local dev env.
```
docker exec -it netdata netdata-claim.sh -token=... -verbose -url=https://traefik/ -insecure -id=7
```
* Check that nodes running on the host can be claimed with the correct user.
* Check that permission errors in the claim directory cause the claim to abort with an error (e.g. install under a normal user account and chown claim.d to root).
* Check that the ACLK starts automatically after claiming.


##### Additional Information
